vllm/platforms/cpu.py

```diff
@@ -65,6 +68,12 @@ def check_and_update_config(cls, vllm_config: VllmConfig) -> None:
         if not model_config.enforce_eager:
             model_config.enforce_eager = True
 
+        if model_config.enable_sleep_mode:
+            logger.warning("sleep mode is not supported on CPU, disable it.")
+            model_config.enable_sleep_mode = False
+
+        model_config.disable_cascade_attn = True
```

Comment:
This is new?

Reply:
Yes I think the cascade attn is only supported in the flash attn backend so I disable it here.

I noticed ```support_sleep_mode``` has became a platform attribute, remove the checking here.